### PR TITLE
Add inline-image style for BPMN icon images

### PR DESCRIPTION
**Pull request by mesellings**
## Description

Closes https://github.com/camunda/camunda-docs/issues/4629.

This is a quick fix that applies an inline style to these BPMN icon images to at least make them smaller and remove borders, padding, and so on.

However, some of these icons already have vertical spacing within the SVG, and we shouldn't changes these icons without understanding the impact on wherever else these are used in the docs. As a result, the gateway icon does not 'sit' on the baseline.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [X] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [X] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Review by akeller — DISMISSED** [review]
Looks good when running locally! Thank you!

BTW, I saw you removed "like" from the sentence. I _think_ this is trying to communicate that it's an example gateway symbol as there are multiple.

**Comment by mesellings** [comment]
Hi @camunda/tech-writers please can I get an approval as Amara is away this week? I just added the word "such as" back in after prior approval.

**Review by conceptualshark — APPROVED** [review]
No idea why these don't build in the preview, but the images do look great locally -thanks for this fix!

**Pull request merged**
